client/my-sites/email/inbox/mailbox-selection-list/style.scss

```
@@ -0,0 +1,196 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+
+body {
```

This selector needs to be made more specific -- this will apply to the `body` element _everywhere_ in Calypso once this CSS has been loaded.

I think we should be able to use `body.is-section-inbox`.
```suggestion
body.is-section-inbox {
```

---

Useful, thanks. My thinking was that, the styles are always applied when sections are reloaded.